resources/dicts/relationship_events/normal_interactions/dislike/decrease.json

```
@@ -1,109 +1,151 @@
-		"interactions": [
-			"m_c covered for r_c on something minor.",
-			"m_c cheers r_c up after a small disagreement.",
-			"m_c, while grabbing food for {PRONOUN/m_c/self}, saw r_c grabbing a really tasty looking rabbit and asked to share it with {PRONOUN/r_c/object}."
-		],
-		"reaction_random_cat": {
-			"dislike": "decrease"
-		}
-	},
-	{
-		"id": "dislike_de_med3",
-		"interactions": [
-            		"m_c recently began looking forward to the little arguments {PRONOUN/m_c/subject} had with r_c.",
-            		"m_c and r_c's bickering has gotten more playful as of late.",
-            		"m_c was almost sad {PRONOUN/m_c/subject} didn't have a chance to argue with r_c today.",
-            		"Even though r_c didn't like m_c, {PRONOUN/r_c/subject} didn't chase {PRONOUN/m_c/object} away when m_c sat down beside {PRONOUN/r_c/object}.",
-            		"m_c and r_c, despite never getting along, put aside their differences and worked together surprisingly well during an emergency.",
-            		"m_c recently has grown tired of disliking r_c."
-		],
-        	"relationship_constraint": [
-			"dislike_20"
-		]
-	},
-	{
-		"id": "dislike_de_high1",
-		"intensity": "high",
-		"interactions": [
-			"m_c is surprised to hear r_c express an ideal {PRONOUN/m_c/subject} can agree with.",
-			"m_c and r_c realize they have more in common than previously thought.",
-			"m_c actually stood up for r_c when {PRONOUN/r_c/subject} {VERB/r_c/were/was} being scolded for something too harshly."
-		]
-	},
-	{
-		"id": "dislike_de_high2",
-		"intensity": "high",
-		"interactions": [
-			"m_c and r_c joke around about how bad the other Clans smell.",
-			"r_c can't help but laugh when m_c tells a funny joke.",
-			"r_c thinks that m_c isn't so bad after all."
-		],
-		"reaction_random_cat": {
-			"dislike": "decrease"
-		}
-	},
-	{
-		"id": "dislike_de_high3",
-		"intensity": "high",
-		"interactions": [
-			"m_c and r_c heckled another Clan at the Gathering together."
-		],
-		"reaction_random_cat": {
-			"dislike": "decrease"
-		},
-		"main_status_constraint": [
-			"elder",
-			"apprentice",
-			"warrior",
-			"mediator apprentice",
-			"mediator",
-			"deputy",
-			"leader"
-		],
-		"random_status_constraint": [
-			"elder",
-			"apprentice",
-			"warrior",
-			"mediator apprentice",
-			"mediator",
-			"deputy",
-			"leader"
-		]
-	}
-]
+[
+	{
+		"id": "dislike_de_low1",
+		"intensity": "low",
+		"interactions": [
+			"m_c is surprised by r_c being thoughtful.",
+			"m_c and r_c agree about something trivial.",
+			"m_c didn't expect r_c to be so amicable today.",
+			"m_c realizes that {PRONOUN/m_c/subject} misunderstood something r_c said.",
+			"m_c is too tired to be angry at r_c today, and learns something good about {PRONOUN/r_c/object} in the process.",
+			"m_c stops and thinks before assuming the worst of r_c.",
+			"When getting something to eat, m_c sees that r_c left behind {PRONOUN/r_c/poss} favorite prey for someone else to take.",

```

```suggestion
			"When getting something to eat, m_c sees r_c allow someone else to take {PRONOUN/r_c/poss} favorite prey",

```
Slight rewording to clear up ambiguity.  Previous version could be misread as r_c leaving m_c's favorite prey behind for someone else (not m_c) to take, which is opposite from the intention.